assets/js/components/Work/UIWorkListItem.jsx

```diff
@@ -0,0 +1,75 @@
+import React from "react";
+import PropTypes from "prop-types";
+import { Link } from "react-router-dom";
+import { setVisibilityClass } from "../../services/helpers";
+
+const WorkListItem = ({ work }) => {
+  const fileSetsToDisplay = 5;
+
+  return (
+    <>
+      <article
+        className="media box is-shadowless"
+        data-testid="ui-worklist-item"
+      >
+        <figure className="media-left">
+          <p className="image is-128x128">
+            <Link to={`/work/${work.id}`}>
+              <img
+                src={`${work.representativeImage}/full/1280,960/0/default.jpg`}
+                alt={work.title}
+                data-testid="image-work"
+                onError={(e) => {
```

Comment:
I'm not sure `onError()` will ever get hit, and if it does its by accident.   Let's move this conditional logic into the `scr` attribute above.